src/Api/Controllers/AuthController.php

```diff
@@ -61,16 +81,59 @@ protected function sendEmail(BakaUsers $user, string $type): void
                 $body = sprintf(_('Your password was update please, use this link to activate your account: %sActivate account%s'), '<a href="' . $activationUrl . '">', '</a>');
                 // send email that password was update
                 break;
+            case 'email-change':
+                $emailChangeUrl = $this->config->app->frontEndUrl . '/user/' . $user->user_activation_email . '/email';
+                $subject = _('Email Change Request');
+                $body = sprintf(_('Click %shere%s to set a new email for your account.'), '<a href="' . $emailChangeUrl . '">', '</a>');
+                break;
             default:
                 $send = false;
                 break;
         }
+
         if ($send) {
             $this->mail
-                ->to($user->email)
-                ->subject($subject)
-                ->content($body)
-                ->sendNow();
+            ->to($user->email)
+            ->subject($subject)
+            ->content($body)
+            ->sendNow();
+
+            print_r('hello');
```

Comment:
@rwhite27 ? Die and print-r?

Reply:
No response

Reply:
Esa funcion esta sin response. Dice que retorna void.